From: Gunnar Wolf <gwolf@debian.org>
Origin: other, http://vmlinux.org/cgi-bin/dwww/usr/share/doc/autotools-dev/README.Debian.gz
Bug-Debian: 549965
Forwarded: no
Last-Update: 2009-12-03
Description: Execute systemwide config.{guess,sub} instead of upstream-provided
 To avoid the outdated GNU config.{sub,guess} problem (see section of that 
 name refered in the "origin" field), we execute the systemwide, updated 
 version. The applied solution here applies Bill Allombert's idea mentioned 
 in said URL.

Index: cherokee-0.99.41/config.guess
===================================================================
--- cherokee-0.99.41.orig/config.guess	2010-01-27 12:05:31.000000000 -0600
+++ cherokee-0.99.41/config.guess	2010-01-27 12:17:06.000000000 -0600
@@ -1,4 +1,9 @@
 #! /bin/sh
+# Execute the systemwide config.guess instead if available (thanks to 
+# Brad Smith for Debian bug #549965, Bill Allombert for generic solution)
+if [ -x /usr/share/misc/config.guess ]; then
+    exec /usr/share/misc/config.guess "$@"
+fi
 # Attempt to guess a canonical system name.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009
Index: cherokee-0.99.41/config.sub
===================================================================
--- cherokee-0.99.41.orig/config.sub	2010-01-27 12:05:31.000000000 -0600
+++ cherokee-0.99.41/config.sub	2010-01-27 12:17:06.000000000 -0600
@@ -1,4 +1,9 @@
 #! /bin/sh
+# Execute the systemwide config.guess instead if available (thanks to 
+# Brad Smith for Debian bug #549965, Bill Allombert for generic solution)
+if [ -x /usr/share/misc/config.sub ]; then
+    exec /usr/share/misc/config.sub "$@"
+fi
 # Configuration validation subroutine script.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009
